Makefile

```diff
@@ -421,10 +421,10 @@ delete_all_virsh_resources: destroy_all_nodes delete_minikube kill_all_port_forw
 	skipper run $(SKIPPER_PARAMS) 'discovery-infra/delete_nodes.py -ns $(NAMESPACE) -a'
 
 download_service_logs:
-	./scripts/download_logs.sh download_service_logs
+	JUNIT_REPORT_DIR=$(REPORTS) ${DEBUG_FLAGS} python3 ./scripts/download_logs.sh download_service_logs
 
 download_cluster_logs:
-	./scripts/download_logs.sh download_cluster_logs
+	JUNIT_REPORT_DIR=$(REPORTS) DEBUG_FLAGS="$(DEBUG_FLAGS)" ./scripts/download_logs.sh download_cluster_logs
```

Comment:
why the difference between the lines?
``${DEBUG_FLAGS}`` vs ``DEBUG_FLAGS="$(DEBUG_FLAGS)"``
also ``python3``

Reply:
By mistake. Fixed